src/main/java/org/apache/accumulo/testing/performance/tests/ConditionalMutationsPT.java

```diff
@@ -93,34 +93,37 @@ private static void runConditionalMutationsTest(Environment env, String tableNam
     ConditionalWriter cw = env.getClient().createConditionalWriter(tableName,
         new ConditionalWriterConfig());
 
+    // warm-up run
     conditionalMutationsTime(cw, 0);
 
     double rateSum = 0.0;
-    for (long i = 1; i < 20; i++) {
+    for (long i = 1; i < 21; i++) {
```

Comment:
I think it might be intuitive if `<=` were used:
```suggestion
    for (long i = 1; i <= 20; i++) {
```

Reply:
Good idea, definitely more intuitive.